### PR TITLE
Release 4.0.2

### PR DESCRIPTION
**Pull request by mrotrebor**
Updated spotbugs 4.1.4

### CODE REVIEW

**Review by KengoTODA — CHANGES_REQUESTED** [review]
Could you bump up `findsecbugs` too? It [released an update](https://github.com/find-sec-bugs/find-sec-bugs/releases/tag/version-1.11.0) recently.

Also need to [generate XML files](https://github.com/spotbugs/sonar-findbugs/blob/e2b707513a8cbfbb4a026afdfc4cfdbe79def6c5/generate_profiles/README.md) to follow update in spotbugs and its plugin.

**Comment by mrotrebor** [comment]
I will add findsecbugs too. 
I already did XML files generation, but there wasn't any changes. I will check now with the findsecbugs.

File: README.md [hunk 1]
```diff
@@ -41,4 +41,5 @@ Findbugs Plugin version|Embedded SpotBugs/Findbugs version|Embedded Findsecbugs
 3.11.0                 | 3.1.12 (SpotBugs)                | 1.8.0                      | 7.4.3sb                   | 1.8|7.6~|5.10.1.16922
 4.0.0                  | 4.0.0 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
 4.0.1                  | 4.1.2 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
-4.0.2-SNAPSHOT         | 4.1.2 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
\ No newline at end of file
+4.0.2                  | 4.1.4 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
+4.0.3-SNAPSHOT         | 4.1.4 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
```
Comment (KengoTODA) [inline README.md:45]: please change `1.10.1` to `1.11.0`

File: README.md [hunk 2]
```diff
@@ -41,4 +41,5 @@ Findbugs Plugin version|Embedded SpotBugs/Findbugs version|Embedded Findsecbugs
 3.11.0                 | 3.1.12 (SpotBugs)                | 1.8.0                      | 7.4.3sb                   | 1.8|7.6~|5.10.1.16922
 4.0.0                  | 4.0.0 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
 4.0.1                  | 4.1.2 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
-4.0.2-SNAPSHOT         | 4.1.2 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
\ No newline at end of file
+4.0.2                  | 4.1.4 (SpotBugs)                 | 1.10.1                     | 7.4.7 (sb-contrib)        | 1.8|7.6~|5.10.1.16922
```
Comment (KengoTODA) [inline README.md:44]: Sorry I forgot requesting you to handle, but the version of `findsecbugs` for `sonar-findbugs 4.0.2` is needed to be updated too.

Comment (mrotrebor) [reply README.md:44]: No problem. I'm not a maven expert, but I think it's done.

**Review by KengoTODA — APPROVED** [review]
Thanks, I will release 4.0.2 after I merge this PR and #332.

**Pull request merged**